cf-oscore/src/main/java/org/eclipse/californium/oscore/Decryptor.java

```diff
@@ -256,14 +256,14 @@ private static void decodeObjectSecurity(Message message, Encrypt0Message enc) t
 		if (h != 0) {
 			int s = total[index];
 
-			contextID = Arrays.copyOfRange(total, index + 1, index + 1 + s);
+			kidContext = Arrays.copyOfRange(total, index + 1, index + 1 + s);
 
 			index += s + 1;
 
 			if (s > 0) {
 				System.out.print("Received KID Context: 0x");
-				for(int i = 0 ; i < contextID.length ; i++) {
-					System.out.print(String.format("%02X", contextID[i])); }
+				for(int i = 0 ; i < kidContext.length ; i++) {
+					System.out.print(String.format("%02X", kidContext[i])); }
 				System.out.println("");
```

Comment:
May be the `System.out.print` could be converted into using the `LOGGER` ?

Reply:
Changed in commit e317f353f957d83c2a362aa930a4ab7875fa4229